docs/features/aws-iam/tutorials/aws-visibility.mdx

```diff
@@ -0,0 +1,180 @@
+* Automatically detect and view the Lambda function calls in Otterize.
+
+By the end, you'll know how to map Kubernetes workloads alongside their dependent AWS resources using Otterize.
+
+## Prerequisites
+
+### CLI tools
+We will need the following CLI tools to set up our cluster and deploy our scripts.
+
+1. [AWS CLI](https://docs.aws.amazon.com/cli/latest/userguide/getting-started-install.html). You will also need credentials within the target account with permissions to work with EKS clusters, IAM, CloudFormation, and Lambda functions.
+2. [eksctl](https://eksctl.io/installation/)
+
+### Create an EKS cluster
+Already have Otterize deployed with the IAM integration configured on your cluster? [Skip to the tutorial.](#tutorial)
+
+Begin by creating an EKS cluster for pod deployment using **eksctl** with the YAML configuration below:
+```bash
+curl ${ABSOLUTE_URL}/code-examples/aws-visibility/eks-cluster.yaml | eksctl create cluster -f -
+```
+<details>
+    <summary>Inspect eks-cluster.yaml contents</summary>
+
+```yaml
+{@include: ../../../../static/code-examples/aws-visibility/eks-cluster.yaml}
+```
+</details>
+
+Next, update **kubeconfig** to link it with the new cluster:
+```bash
+aws eks update-kubeconfig --name otterize-tutorial-aws-visibility --region 'us-west-2'
+```
+
+### Enable AWS Visibility with Otterize Installation
+To provide visibility, we will need to install Otterize in our cluster, and we will want to enable AWS IAM roles for service accounts (IRSA) on our cluster. We can quickly enable this using a cloudformation template on the Otterize Cloud Integrations page.
+
+1. **Install Otterize**
+If you don't have a connected Kubernetes cluster, create one via [Integrations page](https://app.otterize.com/integrations) and follow the setup instructions for Kubernetes. Skip if your cluster is already connected.
+
+2. **Integrate AWS with Otterize Cloud**
+To begin the integration with AWS, visit the [Integrations page](https://app.otterize.com/integrations). Once there, you will be asked for information to help populate a CloudFormation template we will use to set up roles and policies for the Otterize deployment in our cluster.
+
+If you created the EKS cluster above, the cluster name would be￼`otterize-tutorial-aws-visibility`, and the region would be `us-west-2`.
+
+Once the information is provided, a *Launch Cloudformation* button will take you to the AWS Console to deploy the cloudformation script. This script will install IRSA within your EKS cluster and enable Otterize Cloud to manage intents.
+
+After IRSA is enabled in your cluster, you need to redeploy Otterize with the AWS credential operator and AWS visibility enabled. In Otterize Cloud, click the *Next* button to see the updated Helm commands. AWS Visibility is not enabled by default. Before executing the revised configuration, you will need to set an additional flag at the end of the command:
```

Comment:
I know I probably asked for the tutorial to be this way, but having read it now, I realize it would be a lot more powerful if the tutorial didn't ask you to set up Otterize AWS IAM at all! But deployed just the network mapper with AWS IAM visibility, showed you what you can see, and then showed you using the access graph that you can now generate the ClientIntents automatically.

Could you change it so that the IAM policies that are created using the ClientIntents now, are created by the CloudFormation that provisions the resources, so that this tutorial no longer requires the intents operator?

Then, the setup should be:
1. Connect cluster to Otterize Cloud via Kubernetes integration (no AWS integration).
2. Add AWS visibility flag.
3. Done!
4. See AWS connections in the graph
5. See AWS generated intents (YAMLs) in Otterize Cloud.
6. Ready for what's next? Use Otterize to automate AWS IAM, check the other tutorial -->